gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json

```diff
@@ -1,10 +1,11 @@
 [
-{
-  "id": "mon_zombie_brainless",
-    "fg": [
-      { "weight": 100, "sprite": "zombie_headless_f" },
-      { "weight": 100, "sprite": "zombie_headless_m" }
-    ],
-  "bg": "shadow_32x64_normal"
-}
-]
+  {
+    "id": "mon_zombie_brainless",
+    "fg": [ { "weight": 100, "sprite": "zombie_headless_f" }, { "weight": 100, "sprite": "zombie_headless_m" } ],
```

Comment:
I guess it was changed automatically by some linter? Can you please revert joining this line as it is less readable now?